buildinglocally.md

```diff
@@ -334,10 +334,9 @@ cd ..
 # build and install nd4j to maven locally
 git clone https://github.com/deeplearning4j/nd4j.git
 cd nd4j
-mvn clean install -DskipTests -Dmaven.javadoc.skip=true -pl '!:nd4j-cuda-7.5,!:nd4j-cuda-7.5-platform,!:nd4j-tests'
-## More recent 0.6.1 version of the above command
-mvn clean install -DskipTests -Dmaven.javadoc.skip=true -pl '!:nd4j-cuda-8.0,!:nd4j-cuda-8.0-platform,!:nd4j-tests'
-
+mvn clean install -DskipTests -Dmaven.javadoc.skip=true -pl '!:nd4j-cuda-9.0,!:nd4j-cuda-9.0-platform,!:nd4j-tests'
+# or cross-build across Scala versions
+# bash buildmultiplescalaversions.sh clean install -DskipTests -Dmaven.javadoc.skip=true -pl '!:nd4j-cuda-9.0,!:nd4j-cuda-9.0-platform,!:nd4j-tests'
```

Comment:
Cross-building should really be the uncommented default.

Reply:
Agreed and changed.